src/sentry/utils/appleconnect/__init__.py

```diff
@@ -0,0 +1,86 @@
+import datetime
+from collections import namedtuple
+from typing import Optional
+
+import requests
+
+from sentry.models.project import Project
+from sentry.utils import fernet_encrypt as encrypt
+from sentry.utils import json
+
+from . import appstore_connect, itunes_connect
+
+# We don't know the validity for sure, but we estimate 2 weeks
+TOKEN_VALIDITY = datetime.timedelta(weeks=2)
+
+AppConnectCredentialValidity = namedtuple(
+    "AppConnectCredentialValidity",
+    ["appstore_credentials_valid", "itunes_session_valid", "expiration_date"],
+)
+
+
+def credentials_key_name():
+    return "sentry:appleconnect_key"
+
+
+def symbol_sources_prop_name():
+    return "sentry:symbol_sources"
```

Comment:
I wouldn't put it here ( in my mind at least) this is a lower level util that doesn't really deal with sentry storage (prop names... ).